src/main/java/com/palantir/weblogger/WebLoggerResource.java

```diff
@@ -55,54 +54,29 @@ public WebLoggerResource(WebLoggerConfiguration config) {
     }
 
     @POST
-    public void logContent(String eventJsonString) throws ParseException {
+    @Path("{eventName}")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public void logContent(String jsonStringEvent, @PathParam("eventName") String eventName) throws ParseException {
```

Comment:
also prefer the params to be the other way around (feel free to ignore this comment)